src/main/io/beeper.c

```diff
@@ -338,11 +340,20 @@ void beeperUpdate(timeUs_t currentTimeUs)
         return;
     }
 
+    #ifdef USE_DSHOT
+    if (!ARMING_FLAG(ARMED) && currentBeeperEntry->mode == BEEPER_RX_SET) {  
```

Comment:
The loss of transmitter, blackbox erase completed, gyro calibration, acc calibration, maybe some others might be good ones to add in the future with different beep sounds... but that can be a separate PR .